.github/workflows/test.yml

```diff
@@ -162,7 +162,6 @@ jobs:
       - name: Install dependencies and nothing else
         run: |
           conda install setuptools-rust
-          conda install pyarrow>=4.0.0
```

Comment:
is the pyarrow requirement no longer needed?

Reply:
Thankfully no. I found a portion of the DataFusion library where those mappings were already present so I just wrote a Rust function that I exposed to Python that allowed me to use that instead of including PyArrow in the Python layer since DataFusion already includes Arrow.

Previously I didn't know that portion of the DataFusion library existed so that was the reason for the original inclusion of PyArrow.